packages/rum/src/boot/recorderApi.ts

```diff
@@ -63,6 +63,11 @@ export function makeRecorderApi(startRecordingImpl: StartRecording): RecorderApi
         if (state.status === RecorderStatus.IntentToStart) {
           startStrategy()
         }
+
+        if (!session.hasReplayPlan() && state.status === RecorderStatus.Started) {
```

Comment:
```suggestion
        if (state.status === RecorderStatus.Started) {
```
that does not break any test

Reply:
In the current implementation, we only want to stop when the session becomes untracked or lite.
I've updated the tests